CMakeLists.txt

```diff
@@ -156,4 +156,8 @@ if (HAVE_OPM_DATA)
         opm_add_python_test( check_INIT_NORNE ${PROJECT_SOURCE_DIR}/tests/compare_INIT.py $<TARGET_FILE:flow> ${OPM_DATA_ROOT}/norne/NORNE_ATW2013.DATA ${OPM_DATA_ROOT}/norne/ECL.2014.2/NORNE_ATW2013.INIT PORO PORV PERMX )
     endif()
 
+		if(HAVE_OPM_OUTPUT)
```

Comment:
Currently `opm-output`  is a hard dependency for `opm-simulators`, and I guess there are no plans to change that - so I would suggest dropping the `if (HAVE_OPM_OUTPUT)`  test completely.